### DISCUSSION TITLE
Why is RHVAE eval loss negative?

### DISCUSSION THREAD

**Discussion by axu-git** [clementchadebec/benchmark_VAE#130] [General]
I am running the RHVAE google colab code using a Jupyter notebook on my desktop computer, except with the following changes:
```
num_train = 10000
num_eval = 2500
train_dataset = mnist_trainset.data[0:num_train].reshape(-1, 1, 28, 28) / 255.
eval_dataset = mnist_trainset.data[num_train:num_train + num_eval].reshape(-1, 1, 28, 28) / 255.
...
num_epochs=25
```
to reduce the runtime.

I see something like:
```
Train loss: 29.2248
Eval loss: -3.1403
```

Why is the eval loss negative?

Thanks!
